apypie/api.py

```diff
@@ -72,7 +73,7 @@ def __init__(self, **kwargs):
         self.apidoc_cache_dir = kwargs.get('apidoc_cache_dir', apidoc_cache_dir_default)
         self.apidoc_cache_name = kwargs.get('apidoc_cache_name', self._find_cache_name())
 
-        self._session = requests.Session()
+        self._session = kwargs.get('session') or requests.Session()
```

Comment:
Do you use `or` here to make it lazy? I think this would always instantiate a session, even if one was passed.
```suggestion
        self._session = kwargs.get('session', requests.Session())
```

Reply:
Nah, mostly because I am lazy myself ;)

Reply:
But you're right, the one is lazy, the other is not:

```python
>>> a ={'lol': 'what'}
>>> a.get('lol', whaaaa)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'whaaaa' is not defined
>>> a.get('lol') or whaaaa
'what'
```